gcalbot/gcalbot/http.go

```diff
@@ -313,6 +313,25 @@ func (h *HTTPSrv) configHandler(w http.ResponseWriter, r *http.Request) {
 	if calendarID == previousCalendarID {
 		h.Stats.Count("config - update")
 
+		if (!page.Invite && page.Reminder == "") && (inviteInput != "" || reminderInput != "") {
+			// this update must open a new webhook channel, do that now and if it errors, fail early
+			err = h.handler.createEventChannel(selectedAccount, calendarID)
+			if err != nil {
```

Comment:
can eliminate a level of nesting by having a `case nil:` in the switch below which noops and then return in the `default` case